recipes/deepinv/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "deepinv" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepinv-{{ version }}.tar.gz
+  sha256: 1b44b473544e6f2998e7f80dbcb94136a3528e06dfa53b5c00c2d4c2d0bc4ec1
+
+build:
+  skip: true  # [win]
+  skip: true  # [osx]
+  skip: true  # [py<38]
```

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows. 

```suggestion
  noarch: python
```